internal/pkg/describe/status_describe.go

```diff
@@ -193,10 +213,25 @@ func (s *ecsStatusDescriber) ecsServiceAutoscalingAlarms(cluster, service string
 	if err != nil {
 		return nil, fmt.Errorf("retrieve auto scaling alarm names for ECS service %s/%s: %w", cluster, service, err)
 	}
-	alarms, err := s.cwSvcGetter.AlarmStatus(alarmNames)
+	alarms, err := s.cwSvcGetter.AlarmStatuses(cloudwatch.WithNames(alarmNames))
 	if err != nil {
 		return nil, fmt.Errorf("get auto scaling CloudWatch alarms: %w", err)
 	}
+	for i := range alarms {
+		alarms[i].Type = autoscalingAlarmType
+	}
+	return alarms, nil
+}
+
+func (s *ecsStatusDescriber) ecsServiceRollbackAlarms(app, env, svc string) ([]cloudwatch.AlarmStatus, error) {
+	// This will not fetch imported alarms, as we filter by the Copilot-generated prefix of alarm names. This will also not fetch Copilot-generated alarms with names exceeding 255 characters, due to the balanced truncating of `TruncateAlarmName`.
```

Comment:
[`TruncateAlarmName`](https://github.com/aws/copilot-cli/blob/dffeb8d52b66659e7eb81bfb194846af233ca6c7/internal/pkg/template/workload.go#L526) seems to produce deterministic results. Is is possible for us to fetch the managed rollback alarms by their exact names?

Reply:
Part of the problem is that the type of alarm is included in the alarm name, so if we're fetching without a specific alarm type, we can't truncate in the same way as `TruncateAlarmName`. I assume very few alarm names will actually require truncation, as 255 characters is quite long.